.github/workflows/lighthouse-ci.yml

```diff
@@ -0,0 +1,46 @@
+name: Lighthouse CI
+on: [push]
+env:
+    HUGO_VERSION: 0.67.1
+    NODE_VERSION: 10
```

Comment:
You can use a combination of the [`engines` property in `package.json`](https://docs.npmjs.com/files/package.json#engines) and an `.npmrc` to select and enforce using a particular node version, so we could try that.

For Hugo, I'm not sure...is there a convention? It looks like [Netlify uses an environment variable](https://community.netlify.com/t/how-do-i-change-the-version-of-hugo-used-to-build-my-site/17883). We could try creating a `.env` file that gets `source`d, or I think there are a bunch of fancier solutions for automatically setting up an environment too.

Reply:
An env file that gets sourced seems reasonable, but let's not put effort into anything fancy  — minimum solution could just be to document it so we remember to change both. I don't have a sense yet of how often we'll want or need to change it

Reply:
Makes sense. We have some language in the README now about checking the badge to make sure you're using the right version of hugo, so perhaps it could go with that.

What about if we create an Asana template task for publishing issues (like we do for software releases), and that task includes updating Hugo to latest version and then committing the change here (and on the README badge) if tests pass?

Reply:
Created an Asana template task with instructions for updating the Hugo version we test against/build with as part of issue releases.